config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Kaminari.configure do |config|
+  config.default_per_page = 2
```

Comment:
This is set low for development/demo purposes?

Reply:
Yes, and apparently I forgot to undo it.